test/WormholeL2VoteAggregator.t.sol

```diff
@@ -163,6 +163,32 @@ contract CastVote is L2VoteAggregatorTest {
     l2VoteAggregator.castVote(1, _support);
   }
 
+  function testFuzz_RevertWhen_AfterCastWindow(
+    uint96 _amount,
```

Comment:
why are two params underscore prefix, but the others aren't?

Reply:
Good question, I will giv them all a prefix